packages/lesswrong/components/comments/CommentsItem/CommentsItem.tsx

```diff
@@ -418,6 +419,7 @@ export const CommentsItem = ({ treeOptions, comment, nestingLevel=1, isChild, co
               <div className={classes.usernameSpacing}>○</div>
             }
             {post && <CommentShortformIcon comment={comment} post={post} />}
+            {displayTagIcon && <span>TODO</span>}
```

Comment:
@jpaddison3 Is this related to https://github.com/ForumMagnum/ForumMagnum/pull/6456? Did you mean to fill this in?

Reply:
Yikes